templates/internet-of-things/index.html

```diff
@@ -324,216 +261,231 @@ <h2>Ready&ndash;to&ndash;go gateway solutions</h2>
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/49a1df89-qualcommLOGO.png",
+              alt="",
+              width="475",
+              height="106",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"},
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/4d5b8d51-riscv-color.svg",
+              alt="",
+              width="300",
+              height="48",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"},
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/0ebb345c-MediaTek_logo.svg",
+              alt="",
+              width="1024",
+              height="268",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"},
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/24e12cca-logo-nvidia.svg",
+              alt="",
+              width="296",
+              height="144",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"},
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/e3a042a7-raspberrypi-card.png",
+              alt="",
+              width="444",
+              height="123",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"},
+              ) | safe
+            }}            
+          </div>
+        </div>
+      </div>
     </div>
   </div>
 </section>
 
+<hr class="is-fixed-width">
+
 <section class="p-strip">
   <div class="row">
-    <div class="col-7">
-      <h2>Best for developers</h2>
-      <p>With Ubuntu, you can develop a packaged application on your laptop, then monetise it by publishing it directly to the snap store.</p>
-      <p><a href="https://docs.ubuntu.com/core/en/">Learn about developing your IoT project with Ubuntu&nbsp;&rsaquo;</a></p>
+    <div class="col-6">
+      <h2>IoT professional services</h2>
+      <p>Our expert technical teams offer full enablement, customisation and development to ensure that taking your devices to market is fast, scalable and seamless. You can focus on your applications, we handle the rest, including full support and technical workshops.</p>
+      <p>
+        <a href="/core/services" class="p-button--positive">Learn more </a>
```

Comment:
Left a comment in the doc asking the pdm to make this more descriptive, Ant flagged this in another recent build